packages/graphql/src/toESModules.js

```diff
@@ -0,0 +1,42 @@
+import { EOL } from 'os';
+
+export default function(source) {
```

Comment:
Could you add the function name here so its a bit easier to read on its own? JSDoc would be nice to have.
```suggestion
export default function toESModules(source) {
```